x-pack/solutions/security/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -24,6 +24,8 @@ export const FIND_CSP_BENCHMARK_RULE_API_CURRENT_VERSION = '1';
 
 export const GRAPH_ROUTE_PATH = '/internal/cloud_security_posture/graph';
 export const GRAPH_API_CURRENT_VERSION = '1';
+export const GENERIC_ENTITY_INDEX_ENRICH_POLICY =
+  'entity_store_field_retention_generic_default_v1.0.0';
```

Comment:
`GENERIC_ENTITY_INDEX_ENRICH_POLICY` is not space aware
this will cause us problems
make sure to set it with different spaces as well (I suggest API FTR)